bot/code_review_bot/workflow.py

```diff
@@ -325,6 +369,8 @@ def build_task(self, task_id, task_status):
             return InferTask(task_id, task_status)
         elif name.startswith("source-test-"):
             logger.error(f"Unsupported {name} task: will need a local implementation")
+        elif skip_unknown is True:
+            logger.info("Skipping unknown task", id=task_id, name=name)
         else:
             raise Exception("Unsupported task {}".format(name))
```

Comment:
Maybe we should always skip unknown tasks, but log with INFO level when we're on autoland, WARNING level when we're on try.

On try there could still be unknown tasks, e.g. if a developer adds jobs to the reviewbot's try push.

Reply:
I now manage the log level according to the environment used.